### PR TITLE
PM Deploy CSV updates round 2

### PR DESCRIPTION
**Pull request by rtravis-whoi**
This PR contains the remainder of the PM Deploy CSV updates for:
PMCO, PMCI, PMUO, PMUI
**NOTE - we will need to add 3 MOPAKs to the sensor bulk load csv before pushing this to ooi-integration:
CGINS-MOPAK0-11901
CGINS-MOPAK0-11345
CGINS-MOPAK0-11366

### CODE REVIEW

File: deployment/CP02PMUO_Deploy.csv [hunk 1]
```diff
@@ -1,121 +1,121 @@
...
+AR-34,,AR-39,,CP02PMUO-WFP01-02-DOFSTK000,13,1,2019-04-23T14:11:00,2019-10-14T12:01:00,CGMCP-02PMUO-00013,CGVEH-CWFPCP-10304,CGINS-DOFSTK-02726,39.9417,-70.7804,,425,449,
+AR-34,,AR-39,,CP02PMUO-WFP01-03-CTDPFK000,13,1,2019-04-23T14:11:00,2019-10-14T12:01:00,CGMCP-02PMUO-00013,CGVEH-CWFPCP-10304,CGINS-CTDPFK-00127,39.9417,-70.7804,,425,449,
+AR-34,,AR-39,,CP02PMUO-WFP01-04-FLORTK000,13,1,2019-04-23T14:11:00,2019-10-14T12:01:00,CGMCP-02PMUO-00013,CGVEH-CWFPCP-10304,CGINS-FLORTK-01100,39.9417,-70.7804,,425,449,
+AR-34,,AR-39,,CP02PMUO-WFP01-05-PARADK000,13,1,2019-04-23T14:11:00,2019-10-14T12:01:00,CGMCP-02PMUO-00013,CGVEH-CWFPCP-10304,CGINS-PARADK-20582,39.9417,-70.7804,,425,449,
 AR-39,,,,CP02PMUO-RII01-02-ADCPSL010,14,1,2019-10-14T18:09:00,,CGMCP-02PMUO-00014,,CGINS-ADCPSL-22846,39.941912,-70.779573,,426,449,
 AR-39,,,,CP02PMUO-SBS01-00-STCENG000,14,1,2019-10-14T18:09:00,,CGMCP-02PMUO-00014,,CGCON-BSTC01-00005,39.941912,-70.779573,,0,449,
 AR-39,,,,CP02PMUO-SBS01-01-MOPAK0000,14,1,2019-10-14T18:09:00,,CGMCP-02PMUO-00014,,CGINS-MOPAK0-11358,39.941912,-70.779573,,0,449,
```
Comment (asmith-whoi) [inline deployment/CP02PMUO_Deploy.csv]: See comments above. PMUO 6 STC sensor ID should be 00005, PMUO 10 WFP depths should be 423

File: deployment/CP02PMCO_Deploy.csv [hunk 2]
```diff
@@ -91,7 +100,7 @@ AR-31,,AR-34,,CP02PMCO-WFP01-04-FLORTK000,11,1,2018-11-02T20:28:00,2019-04-23T16
...
 AR-34,,AR-39,,CP02PMCO-RII01-02-ADCPTG010,12,1,2019-04-22T04:09:00,2019-10-08T12:02:00,CGMCP-02PMCO-00012,,CGINS-ADCPTG-19336,40.1012,-70.8877,,128,148,
 AR-34,,AR-39,,CP02PMCO-SBS01-00-STCENG000,12,1,2019-04-22T04:09:00,2019-10-08T12:02:00,CGMCP-02PMCO-00012,,CGCON-BSTC01-00010,40.1012,-70.8877,,0,148,
-AR-34,,AR-39,,CP02PMCO-SBS01-01-MOPAK0000,12,1,2019-04-22T04:09:00,2019-10-08T12:02:00,CGMCP-02PMCO-00012,,CGINS-MOPAK0-99999,40.1012,-70.8877,,0,148,sn not in config
+AR-34,,AR-39,,CP02PMCO-SBS01-01-MOPAK0000,12,1,2019-04-22T04:09:00,2019-10-08T12:02:00,CGMCP-02PMCO-00012,,CGINS-MOPAK0-99999,40.1012,-70.8877,,0,148,
 AR-34,,AR-39,,CP02PMCO-WFP01-00-WFPENG000,12,1,2019-04-22T04:09:00,2019-10-08T12:02:00,CGMCP-02PMCO-00012,CGVEH-CWFPCP-10305,CGCON-CWFPCP-10305,40.1012,-70.8877,,127,148,
 AR-34,,AR-39,,CP02PMCO-WFP01-01-VEL3DK000,12,1,2019-04-22T04:09:00,2019-10-08T12:02:00,CGMCP-02PMCO-00012,CGVEH-CWFPCP-10305,CGINS-VEL3DK-00034,40.1012,-70.8877,,127,148,
 AR-34,,AR-39,,CP02PMCO-WFP01-02-DOFSTK000,12,1,2019-04-22T04:09:00,2019-10-08T12:02:00,CGMCP-02PMCO-00012,CGVEH-CWFPCP-10305,CGINS-DOFSTK-02628,40.1012,-70.8877,,127,148,
```
Comment (asmith-whoi) [inline deployment/CP02PMCO_Deploy.csv]: OK except as noted above, PMCO 4 water depths.

File: deployment/CP02PMCO_Deploy.csv [hunk 3]
```diff
@@ -1,85 +1,94 @@
...
+AT-31,,AT-31,,CP02PMCO-SBS01-01-MOPAK0000,4,1,2015-10-14T19:05:00,2015-10-17T12:25:00,CGMCP-02PMCO-00004,,CGINS-MOPAK0-99999,40.10073,-70.88833,,0,148,
+AT-31,,AT-31,,CP02PMCO-WFP01-00-WFPENG000,4,1,2015-10-14T19:05:00,2015-10-17T12:25:00,CGMCP-02PMCO-00004,CGVEH-CWFPCP-99101,CGCON-CWFPCP-99101,40.10073,-70.88833,,127,148,
+AT-31,,AT-31,,CP02PMCO-WFP01-01-VEL3DK000,4,1,2015-10-14T19:05:00,2015-10-17T12:25:00,CGMCP-02PMCO-00004,CGVEH-CWFPCP-99101,CGINS-VEL3DK-00015,40.10073,-70.88833,,127,148,
+AT-31,,AT-31,,CP02PMCO-WFP01-02-DOFSTK000,4,1,2015-10-14T19:05:00,2015-10-17T12:25:00,CGMCP-02PMCO-00004,CGVEH-CWFPCP-99101,CGINS-DOFSTK-02978,40.10073,-70.88833,,127,148,
+AT-31,,AT-31,,CP02PMCO-WFP01-03-CTDPFK000,4,1,2015-10-14T19:05:00,2015-10-17T12:25:00,CGMCP-02PMCO-00004,CGVEH-CWFPCP-99101,CGINS-CTDPFK-00111,40.10073,-70.88833,,127,148,
+AT-31,,AT-31,,CP02PMCO-WFP01-04-FLORTK000,4,1,2015-10-14T19:05:00,2015-10-17T12:25:00,CGMCP-02PMCO-00004,CGVEH-CWFPCP-99101,CGINS-FLORTK-01029,40.10073,-70.88833,,127,148,
+AT-31,,AT-31,,CP02PMCO-WFP01-05-PARADK000,4,1,2015-10-14T19:05:00,2015-10-17T12:25:00,CGMCP-02PMCO-00004,CGVEH-CWFPCP-99101,CGINS-PARADK-20437,40.10073,-70.88833,,127,148,
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply deployment/CP02PMCO_Deploy.csv]: Ah yes - discussed w Sheri and we used anchor drop location & depth.  Took directly from cruise notebook (have at my desk).

File: deployment/CP02PMCO_Deploy.csv (hunk 2 above)
Comment (rtravis-whoi) [reply deployment/CP02PMCO_Deploy.csv]: Taken from cruise notebook (at my desk)

File: deployment/CP02PMUO_Deploy.csv (hunk 1 above)
Comment (rtravis-whoi) [reply deployment/CP02PMUO_Deploy.csv]: Fixed per A.Smith comments.

File: deployment/CP02PMCO_Deploy.csv (hunk 3 above)
[the remark this answers is not in the document]
Comment (asmith-whoi) [reply deployment/CP02PMCO_Deploy.csv]: changes OK

File: deployment/CP02PMUO_Deploy.csv (hunk 1 above)
Comment (asmith-whoi) [reply deployment/CP02PMUO_Deploy.csv]: changes OK

File: deployment/CP02PMUO_Deploy.csv [hunk 4]
```diff
@@ -1,121 +1,121 @@
...
```
Comment (reedan88) [inline deployment/CP02PMUO_Deploy.csv]: Changes suggested by Allen look good. Everything else looks good.

File: deployment/CP02PMCI_Deploy.csv [hunk 5]
```diff
@@ -1,112 +1,112 @@
...
+AR-34,,AR-39,,CP02PMCI-WFP01-02-DOFSTK000,12,1,2019-04-17T02:25:00,2019-10-07T20:03:00,CGMCP-02PMCI-00012,CGVEH-CWFPCP-10302,CGINS-DOFSTK-02724,40.2267,-70.8886,,103,126,
+AR-34,,AR-39,,CP02PMCI-WFP01-03-CTDPFK000,12,1,2019-04-17T02:25:00,2019-10-07T20:03:00,CGMCP-02PMCI-00012,CGVEH-CWFPCP-10302,CGINS-CTDPFK-00125,40.2267,-70.8886,,103,126,
+AR-34,,AR-39,,CP02PMCI-WFP01-04-FLORTK000,12,1,2019-04-17T02:25:00,2019-10-07T20:03:00,CGMCP-02PMCI-00012,CGVEH-CWFPCP-10302,CGINS-FLORTK-01119,40.2267,-70.8886,,103,126,
+AR-34,,AR-39,,CP02PMCI-WFP01-05-PARADK000,12,1,2019-04-17T02:25:00,2019-10-07T20:03:00,CGMCP-02PMCI-00012,CGVEH-CWFPCP-10302,CGINS-PARADK-20465,40.2267,-70.8886,,103,126,
 AR-39,,,,CP02PMCI-RII01-02-ADCPTG010,13,1,2019-10-07T18:23:00,,CGMCP-02PMCI-00013,,CGINS-ADCPTG-19151,40.226972,-70.877882,,104,126,
 AR-39,,,,CP02PMCI-SBS01-00-STCENG000,13,1,2019-10-07T18:23:00,,CGMCP-02PMCI-00013,,CGCON-BSTC01-00017,40.226972,-70.877882,,0,126,
 AR-39,,,,CP02PMCI-SBS01-01-MOPAK0000,13,1,2019-10-07T18:23:00,,CGMCP-02PMCI-00013,,CGINS-MOPAK0-58478,40.226972,-70.877882,,0,126,
```
[the remark this answers is not in the document]
Comment (cdobs) [reply deployment/CP02PMCI_Deploy.csv]: Looks good to me.

File: deployment/CP02PMCO_Deploy.csv (hunk 3 above)
[the remark this answers is not in the document]
Comment (cdobs) [reply deployment/CP02PMCO_Deploy.csv]: Agreed ^

File: deployment/CP02PMCO_Deploy.csv (hunk 2 above)
Comment (cdobs) [reply deployment/CP02PMCO_Deploy.csv]: Looks good to me

**Pull request merged**